src/app/campaign-details-card/campaign-details-card.component.scss

```diff
@@ -4,8 +4,18 @@
   @include make-card($box-shadow-hard);
 }
 
-.c-icon-lists {
-  @include card-icons;
+.c-icon-list {
+  list-style-type: none;
```

Comment:
We should probably either try to align and keep using the mixin, or get rid of it if it is now only used in 1 place